jest.config.js

```diff
@@ -30,11 +30,11 @@ module.exports = {
   moduleFileExtensions: ['js', 'json', 'ts', 'tsx'],
   moduleNameMapper: {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|svg)$':
-      require.resolve('./testConfig/fileMock'),
+      require.resolve('./jest/fileMock'),
   },
   transform: {
     '\\.css\\.ts$': require.resolve('@vanilla-extract/jest-transform'),
-    '\\.(j|t)sx?$': require.resolve('./testConfig/babelTransform.js'),
+    '\\.(j|t)sx?$': require.resolve('./jest/babelTransform.js'),
```

Comment:
We're missing `cjs` and `mjs` here too 😓

Reply:
Well, well, well...